src/solver/optimisation/opt_export_structure.cpp

```diff
@@ -65,43 +62,31 @@ const std::initializer_list<std::string>& getNames<ExportStructDict>()
 } // namespace Data
 } // namespace Antares
 
-void OPT_ExportInterco(const Antares::Data::Study& study,
```

Comment:
Here we used to pass the whole study. This is unclear as we only need its writer.